substrate/frame/src/lib.rs

```diff
@@ -361,6 +372,12 @@ pub mod runtime {
 		/// Macro to easily derive the `Config` trait of various pallet for `Runtime`.
 		pub use frame_support::derive_impl;
 
+		/// sovereign account ID for a pallet.
+		pub use frame_support::PalletId;
+
+		/// Runtime storage traits amd types.
+		pub use frame_support::storage::{migration::*, KeyPrefixIterator, StoragePrefixedMap};
```

Comment:
```suggestion
		pub use frame_support::storage::{KeyPrefixIterator, StoragePrefixedMap};
```

I don't see why migration items should be in this prelude